package.json

```diff
@@ -39,13 +41,12 @@
         "@types/enzyme": "^3.10.5",
         "@types/jest": "^24.9.0",
         "@types/lodash": "^4.14.150",
-        "@types/react": "^17.0.2",
-        "@types/react-dom": "^17.0.1",
-        "@types/react-redux": "^7.1.16",
+        "@types/react": "18.3.12",
```

Comment:
Why is there a type mismatch between `@types/react` and `react`?

Reply:
I found [this thread](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/43962) about it. The DefinitelyTyped library uses major and minor versions that match the corresponding library but varies on patch version.